src/main/java/frc/robot/subsystems/shooter/ShooterCommands.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.shooter;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.Command;
+
+/** Add your docs here. */
+public class ShooterCommands extends Command{
+    private final Shooter shooter;
+
+    public ShooterCommands(Shooter shooter) {
+        this.shooter = shooter;
+        this.addRequirements(shooter);
+    }
+
+    public Command reachSpeed(double targetSpeedRPM, PIDController pid) {
```

Comment:
RPM and voltage are not. It means that if you input RPM and try to get voltage as output, it will cause problems. The clamp with voltage makes no sense.

Reply:
הקשר בין חשמל לסיבובים בדקה הוא ישיר (מדובר בפונקציה קווית), אבל זה לא אומר שהם שווים זה לזה. אם רוצים להשתמש בקשר הזה, צריך ליצור 2 משתנים ב-constants, שמייצגים את m ו-b של הפונקציה, ואז להכפיל ולהוסיף כל פעם שרוצים להעביר בין סיבובים לחשמל

Reply:
זה בסדר מכיוון שה PID מקבל speedRPS ומחזיר voltage.